io.c

```diff
@@ -8370,29 +8378,48 @@ io_puts_ary(VALUE ary, VALUE out, int recur)
 
 /*
  *  call-seq:
- *     ios.puts(obj, ...)    -> nil
+ *    puts(*objects) -> nil
+ *    puts           -> newline
```

Comment:
`puts` without arguments returns `nil` (it prints a newline, but it does not return a newline).

Reply:
Fixed.